packages/lit/rollup.config.js

```diff
@@ -58,5 +58,20 @@ export default litProdConfig({
       output: 'lit.min',
       name: 'Lit',
     },
+    {
+      file: 'index',
+      output: 'lit.min.esm',
+      format: 'es',
+    },
+    {
+      file: 'index.max',
+      output: 'lit.max',
```

Comment:
what format is this? Let's add a suffix for this one.

Reply:
The `output` field of each of these will get `.js` appended by `litMonoBundleConfig`, here: https://github.com/lit/lit/blob/81cc2aa0e825fafa49b8f1cce7a784123714835e/rollup-common.js#L434

Before these changes, we were only generating `lit.min.js`. (I don't think this file was ever part of the npm package, so we can probably rename it safely.) After, the bundles are now `lit.min.js`, `lit.min.esm.js`, `lit.max.js`, and `lit.max.esm.js`. You mentioned not wanting to use `.esm` above, is that because you were thinking it was the final extension (i.e. no `.js`) or would you prefer the ESM bundles to be the no-indication default and maybe have a `.umd` infix for the UMD ones intead?

Reply:
yes, I meant no suffix before the extension for standard modules and a `umd` suffix for this bundle. Taking it to the main thread though.